test/scripts/base-host-check.sh

```diff
@@ -78,6 +79,22 @@ get_oscap_score() {
     fi
 }
 
+check_ca_cert() {
+    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
+
+    echo "📗 Checking CA cert anchor file"
+    if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
+        echo "Anchor CA file does not exist"
+        exit 1
+    fi
+
+    echo "📗 Checking extracted CA cert file"
+    if ! [ -e "/etc/pki/ca-trust/source/extracted/pem/directory-hash/Test_CA_for_osbuild.pem.pem" ]; then
```

Comment:
@lzap I just noticed a potential typo in the filename - note the double `.pem.pem` :thinking:

Reply:
Yeah, this is curious, do we generate funky filenames here or is the test not running correctly?

Reply:
@mvo5 see https://github.com/osbuild/images/pull/1076#discussion_r1874828981. The modification is not applied to any image we boot-test, so this code is never executed.